docs/collection-docs.md

```diff
@@ -240,16 +240,6 @@ communication:
   matrix_rooms:
     - topic: General usage and support questions
       room: '#users:ansible.im'
-  irc_channels:
-    - topic: General usage and support questions
-      network: Libera
-      channel: '#ansible'
-  mailing_lists:
-    - topic: Ansible Project List
-      url: https://groups.google.com/g/ansible-project
-      # You can also add a `subscribe` field with an URI that allows to subscribe
-      # to the mailing list. For lists on https://groups.google.com/ a subscribe link is
-      # automatically generated.
```

Comment:
Same here: these need to stay.

Instead of removing them, add comments that indicate that these specific IRC channel and mailing list are only mentioned as examples and should not be used except in very specific circumstances.